mne/io/brainvision/brainvision.py

```diff
@@ -296,9 +298,18 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
                       "not the '%s' file." % ext)
     with open(vhdr_fname, 'rb') as f:
         # extract the first section to resemble a cfg
-        header = f.readline().decode('utf-8').strip()
+        header = f.readline()
+        try:
+            header = header.decode('utf-8').strip()
+        except UnicodeDecodeError:
+            header = header.decode('ISO-8859-1')
```

Comment:
The problem was that you needed to decode before parsing the config, but I've done a semi-parsing with regex assuming ASCII. There shouldn't be anything beyond the header declaration, comment lines and some whitespace, i.e. no relevant non-ASCII text before the codepage declaration, so this  `re.search()` doesn't hit anything it can't handle if it does find something and if not, then the bytes from the other codepages don't match the regex, so still not a problem.